cmd/buildtools/utils.go

```diff
@@ -688,3 +574,14 @@ func RunCommand(cmd *exec.Cmd) error {
 	cmd.Stderr = os.Stderr
 	return cmd.Run()
 }
+
+func NewHelm() (*helm.Helm, error) {
+	k0sVersion, err := GetMakefileVariable("K0S_VERSION")
```

Comment:
should use the existing `getK0sVersion` function since that accounts for `INPUT_K0S_VERSION`

Reply:
done